addons/l10n_ma/data/account_chart_template_data.xml

```diff
@@ -1,8 +1,3814 @@
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_31412" model="account.account.template">
+        <field name="name">Produits intermdiaires (groupe B)</field>
+        <field name="code">31412</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_31451" model="account.account.template">
+        <field name="name">Dchets</field>
+        <field name="code">31451</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_31452" model="account.account.template">
+        <field name="name">Rebuts</field>
+        <field name="code">31452</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_31453" model="account.account.template">
+        <field name="name">Matires de rcupration</field>
+        <field name="code">31453</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3148" model="account.account.template">
+        <field name="name">Autres produits intermdiaires et produits rsiduels</field>
+        <field name="code">3148</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3151" model="account.account.template">
+        <field name="name">Produits finis (groupe A)</field>
+        <field name="code">3151</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3152" model="account.account.template">
+        <field name="name">Produits finis (groupe B)</field>
+        <field name="code">3152</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3156" model="account.account.template">
+        <field name="name">Produits finis en cours de route</field>
+        <field name="code">3156</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3158" model="account.account.template">
+        <field name="name">Autres produits finis</field>
+        <field name="code">3158</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id"
```

Comment:
acomptes verss?  (something missing?)
Could you tell us why you put this liability account as reconcilable?